lib/components/app/app-menu.js

```diff
@@ -41,9 +47,92 @@ class AppMenu extends Component {
     window.location.href = startOverUrl
   }
 
+  _expandedPane = () => {
+    const { isPaneOpenLeft } = this.state
+    this.setState({ isPaneOpenLeft: !isPaneOpenLeft })
+  }
+
+  _expandSubmenu = () => {
+    const { isSubmenuExpanded } = this.state
+    this.setState({ isSubmenuExpanded: !isSubmenuExpanded })
+  }
+
+  _addExtraMenuItems = (menuItems) => {
+    const AddIconAndLabel = ({iconType, iconUrl, label}) => {
+      return (
+        iconUrl
+          ? <span>
+            <img alt={`icon for ${label} Menu Item`} src={iconUrl || 'bus'} />
```

Comment:
We can probably remove the `bus` default, right?

Reply:
Good spot!